src/coriolis.jl

```diff
@@ -0,0 +1,40 @@
+#####
+##### Functions for non-rotating models
+#####
+
+@inline x_f_cross_U(i, j, k, grid::AbstractGrid{T}, ::Nothing, U) where T = zero(T)
+@inline y_f_cross_U(i, j, k, grid::AbstractGrid{T}, ::Nothing, U) where T = zero(T)
+@inline z_f_cross_U(i, j, k, grid::AbstractGrid{T}, ::Nothing, U) where T = zero(T)
+
+"""
+    FPlane{T} <: AbstractRotation
+
+A parameter object for constant rotation around a vertical axis.
+"""
+struct FPlane{T} <: AbstractRotation
+    f :: T
+end
+
+"""
+    FPlane([T=Float64;] f)
+
+Returns a parameter object for constant rotation at the angular frequency
+`2f`, and therefore with background vorticity `f`, around a vertical axis.
+
+Also called `FPlane`, after the "f-plane" approximation for the local effect of 
+Earth's rotation in a planar coordinate system tangent to the Earth's surface.
+"""
+function FPlane(T::DataType=Float64; f) 
```

Comment:
I thought we weren't using types in function signatures unless it's for dispatch? (Following the YASGuide)

Reply:
Hmm... I think this was originally needed for dispatch because `f` came with a default; however it looks like it may not be needed anymore.